src/odata-to-abstract-sql.ts

```diff
@@ -1419,53 +1421,54 @@ export class OData2AbstractSQL {
 	}
 }
 
-const generateShortAliases = (clientModel: AbstractSqlModel) => {
-	const shortAliases: _.Dictionary<string> = {};
-	const addAliases = (origAliasParts: string[]) => {
-		const trie = {};
-		const buildTrie = (aliasPart: string) => {
-			let node: any = trie;
-			for (let i = 0; i < aliasPart.length; i++) {
-				if (node.$suffix) {
-					node[node.$suffix[0]] = {
-						$suffix: node.$suffix.slice(1),
-					};
-					delete node.$suffix;
-				}
-				const c = aliasPart[i];
-				if (node[c]) {
-					node = node[c];
-				} else {
-					node[c] = {
-						$suffix: aliasPart.slice(i + 1),
-					};
-					return;
-				}
-			}
-		};
-		const traverseNodes = (str: string, node: any) => {
+const addAliases = (
+	shortAliases: _.Dictionary<string>,
+	origAliasParts: string[],
+) => {
```

Comment:
I moved this out of the `generateShortAliases` function to avoid shadowed vars, hopefully it might also represent a performance improvement when instantiating odata-to-abstract-sql translators